src/Microsoft.Experimental.Collections/Microsoft/Collections/Extensions/OrderedDictionary.KeyCollection.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Please update header to match
```
// Licensed to the .NET Foundation under one or more agreements.
// The .NET Foundation licenses this file to you under the MIT license.
// See the LICENSE file in the project root for more information.
```

Some of them were wrong a little while ago, maybe you snapped then.